src/plugins/Sharesite/webui/EditToadlet.java

```diff
@@ -128,9 +131,14 @@ private void addNodes( HTMLNode form, String name, String path, String desc, Str
 		insertHour.addChild("br");
 
 		attrs = new String[] { "type", "size", "name", "value" };
-		vals = new String[] { "text",  "2", "insertHourInput",  iHour };
+		vals = new String[] { "text",  "2", "insertHourInput",	iHour };
 		insertHour.addChild("input", attrs, vals);
-		insertHour.addChild("span",l10n.getString("Sharesite.Edit.InsertHourDescription"));
+		Integer currentHour = Calendar.getInstance(TimeZone.getTimeZone("UTC"), Locale.US)
+				.get(Calendar.HOUR_OF_DAY); // 0-23
+		Integer currentMinute = Calendar.getInstance(TimeZone.getTimeZone("UTC"), Locale.US)
+				.get(Calendar.MINUTE); // 0-59
```

Comment:
Ah, yes, I stopped using it in the L10N (because I did not see how to pass it two values) but forgot to remove the instance. Thank you!